src/WebJobs.Extensions.DurableTask/Bindings/BindingHelper.cs

```diff
@@ -29,6 +30,11 @@ public StartOrchestrationArgs JObjectToStartOrchestrationArgs(JObject input, Orc
             return input?.ToObject<StartOrchestrationArgs>();
         }
 
+        public StartOrchestrationArgs StringToStartOrchestrationArgs(string input, OrchestrationClientAttribute attr)
+        {
+            return JsonConvert.DeserializeObject<StartOrchestrationArgs>(input);
```

Comment:
Should we consider handling null input?

Reply:
We should. Making that change and merging.